x-pack/plugins/cases/public/components/insert_timeline/index.tsx

```diff
@@ -11,6 +11,7 @@ import { useTimelineContext } from '../timeline_context/use_timeline_context';
 
 type InsertFields = 'comment' | 'description';
 
+// eslint-disable-next-line react/display-name
```

Comment:
super nit: We can fix this linting error like:

```
export const InsertTimeline: React.FC<{ fieldName: InsertFields }> = ({ fieldName }) => {
  const { setFieldValue, getFormData } = useFormContext();
  const timelineHooks = useTimelineContext()?.hooks;
  const formData = getFormData();
  const onTimelineAttached = useCallback(
    (newValue: string) => setFieldValue(fieldName, newValue),
    [fieldName, setFieldValue]
  );
  timelineHooks?.useInsertTimeline(formData[fieldName] ?? '', onTimelineAttached);
  return null;
};

InsertTimeline.displayName = 'InsertTimeline';
```

Reply:
Fixed 👍